.circleci/config.yml

```
@@ -41,6 +41,9 @@ commands:
       - run:
           name: Print cert hashes for debugging
           command: find tools/ssl -type f -exec md5sum {} \; | sort
+      - run:
+          name: Continue if certificate data is valid
```

The name is a bit misleading as we only check if it exists.
```suggestion
          name: Continue if certificate exists
```